en-US/snippets/2017/mar2017.lang

```diff
@@ -0,0 +1,90 @@
+## NOTE: reference bug 1336204; Deadline: 24 February, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Link for Google Play: https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp
+## NOTE: Link for App Store: https://itunes.apple.com/app/apple-store/id989804926?mt=8
+## NOTE: Campaign: Desktop Snippets
+
+
+## MAX_LENGTH: 120 characters
```

Comment:
As explained in the previous comment, every time you add a metatag to impose a maximum length, you **need** to add a localization comment explaining it. In localized files you'll only see this comment, not the meta tag.

In doubt, look at the previous snippet files you created, e.g. https://github.com/mozilla-l10n/engagement-l10n/blob/master/en-US/snippets/2017/feb2017.lang#L14-L19